### PR TITLE
Move "env status" off of home page

### PR DESCRIPTION
**Pull request by blacktm**
Partners are confused by the current "environments status" on the home page, namely suggesting that they can integrate with multiple login.gov environments. This creates unnecessary support issues. The "environments status" page is still useful, so this commit moves it to a new `/env` path.

### CODE REVIEW

**Comment by jmhooper** [comment]
@blacktm: Should we link to the new environment page somewhere?

File: app/controllers/env_controller.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Controller for the "environments status" page
+class EnvController < ApplicationController
```
Comment (jmhooper) [inline app/controllers/env_controller.rb]: Looks like this controller has no test coverage. We should probably add a controller spec for it.

File: config/routes.rb [hunk 2]
```diff
@@ -8,6 +8,7 @@
   end
   get '/auth/logindotgov/callback' => 'users/omniauth#callback'
   get 'users/none' => 'users#none'
+  get 'env' => 'env#index'
```
Comment (jmhooper) [inline config/routes.rb]: This is a nit, but could we change `get 'env'` to `get '/env'` for consistency.

Comment (blacktm) [reply config/routes.rb]: Updated in 692be98

File: app/controllers/env_controller.rb (hunk 1 above)
Comment (blacktm) [reply app/controllers/env_controller.rb]: Added a controller test in 5db53bd (we have the `DeployStatusChecker` test coverage in `/spec/services/deploy_status_checker_spec.rb` already).

File: spec/controllers/env_controller_spec.rb [hunk 3]
```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+describe EmailsController do
```
Comment (jmhooper) [inline spec/controllers/env_controller_spec.rb]: I think you mean `EnvController` here

Comment (blacktm) [reply spec/controllers/env_controller_spec.rb]: D'oh, fixed, waiting on tests to re-run

**Comment by blacktm** [comment]
@jmhooper Dang it, now getting a `WebMock::NetConnectNotAllowedError`, not sure how to fix. Just needing this "Environments status" page moved so we can stop the support inquiries. Any ideas?

File: spec/controllers/env_controller_spec.rb [hunk 4]
```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+describe EnvController do
+  include Devise::Test::ControllerHelpers
+  WebMock.allow_net_connect!
```
Comment (zachmargolis) [inline spec/controllers/env_controller_spec.rb]: I don't think we should do this .... we should mock the responses we need

File: spec/features/home.rb [hunk 5]
```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+feature 'Home' do
+  WebMock.allow_net_connect!
```
Comment (zachmargolis) [inline spec/features/home.rb]: This is a bad pattern IMO

File: spec/features/home.rb [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+    scenario 'you should see intro and links to dev docs and partners' do
+      visit root_path
+
+      expect(page).to have_content(I18n.t('home.body_intro'))
+      expect(page).to have_content(I18n.t('home.dev_docs_link'))
+      expect(page).to have_content(I18n.t('home.partners_link'))
+    end
+  end
+
+  context 'an adminn user' do
```
Comment (zachmargolis) [inline spec/features/home.rb]: nit: typo in `adminn`

File: spec/support/deploy_status_checker_helper.rb [hunk 7]
```diff
@@ -0,0 +1,29 @@
+module DeployStatusCheckerHelper
```
Comment (zachmargolis) [inline spec/support/deploy_status_checker_helper.rb]: good idea to extract this! Should we also do the same in `deploy_status_checker_spec.rb`'s `#check` section?

**Pull request merged**